lasagne/layers/sparse/dense.py

```diff
@@ -0,0 +1,24 @@
+import theano.sparse as sparse
+
+from ..dense import DenseLayer as Dense
+
+
+__all__ = [
+    "DenseLayer",
+]
+
+
+class DenseLayer(Dense):
+    def get_output_for(self, input, **kwargs):
+        """
+        Parameters
+        ----------
+        input : theano sparse matrix
+            output from the previous layer
+        """
+        assert type(input) == sparse.basic.SparseVariable
+
+        activation = sparse.basic.dot(input, self.W)
```

Comment:
So the default `DenseLayer` will work with sparse input already?

If there are different implementations, then the user should probably be able to choose one in the constructor -- but this is something that could be added later.